pkg/daemon/osrelease/osrelease_test.go

```diff
@@ -132,11 +132,36 @@ VARIANT="CoreOS"
 VARIANT_ID=coreos
 OSTREE_VERSION='37.20230126.20.0'`
 
```

Comment:
issue: We should also add a test for Centos Stream 10 since that's what OKD uses. We can get the `/etc/os-release` info by pulling down the OS image for an OKD release and running `cat /etc/os-release`.